ci/images/setup.sh

```diff
@@ -26,6 +26,15 @@ curl -L ${JDK_URL} | tar zx --strip-components=1
 test -f /opt/openjdk/bin/java
 test -f /opt/openjdk/bin/javac
 
+if [[ $1 != $2 ]]; then
```

Comment:
Could we change this to check to see if `$2` has been defined? That would remove the need to change the Dockerfiles for 8, 11, and 15.

Reply:
Yeah, I think I have a way already. Thanks for the suggestion